### DISCUSSION TITLE
Picture of artists with tranparent background

### DISCUSSION THREAD

**Discussion by kp-org** [jcorporation/myMPD#924] [General]
Hi,

not sure if this is the desired behavior, but if a transparent picture is used the background icon is still visible. I would expect that the icon will be replaced completely.

![image](https://user-images.githubusercontent.com/13482381/205869662-4a6d13ce-d79c-42a2-87d1-3981e8457c42.png)

I would expect that the page's background is visible. My pictures are png files always.

**Comment by jcorporation** [comment]
This was intended. It is the simplest solution to show a loading icon until the real cover is loaded.
